lib/spack/spack/directives.py

```diff
@@ -47,14 +47,13 @@ class OpenMpi(Package):
 from spack.resource import Resource
 from spack.version import Version, VersionChecksumError
 
-
 if sys.version_info >= (3, 3):
     from collections.abc import Sequence  # novm
 else:
     from collections import Sequence
 
 
-__all__ = []
+__all__ = ['DirectiveError', 'DirectiveMeta']
```

Comment:
Modules other than `spack.directives` reference these in their API docs, so I had to make them public to get them to link properly.